packages/xo-server/src/api/disk.mjs

```diff
@@ -226,6 +227,12 @@ async function handleImport(req, res, { type, name, description, vmdkData, srId,
           try {
             await vdi.$importContent(vhdStream, { format: diskFormat })
             res.end(format.response(0, vdi.$id))
+            let buffer
+            const CHUNK_SIZE = 1024 * 1024
+            // drain remaining content ( padding .header)
+            do {
+              buffer = await readChunk(part, CHUNK_SIZE)
```

Comment:
Why not `part.resume()` to drain the stream?

Reply:
resume do the same error, my guess is that we need to read all the data before continuing, and I understand taht resume make the stream flow again, but does not wait fir the end 
or maybe we should also wait for an `end` envent